specs/MultiProfile.md

```diff
@@ -328,6 +362,18 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
+  /// After the Delete() call completes, The render process of webviews on this profile will
+  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
+  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
+  /// for more details. The profile directory on disk will be actually deleted when the browser
+  /// process exits. Profile creation will fail if you create a new profile with the same name
```

Comment:
Yes, we could not create a profile with the same name that has been marked as deleted.  The use can add a prefix with global integer increase or something else like GUID etc.

Reply:
I had thought from discussing with you in previous comments that the profile folder will be deleted as soon as it is no longer in use, which may be in the worst case while shutting down the browser process (since then it would no longer be in use). Did I misunderstand previously and it really will only be deleted when closing down the browser process? Or is the text above just slightly wrong and should instead say something like this:

"The profile directory on disk will be actually deleted when it is no longer in use which in the worst case will be when the browser process exits."

Also, if we are able to rename the folder while waiting for deletion this could perhaps improve this case

Reply:
When the Delete API is called, it will try to delete at the time, but most situation it will fail for file occupied.  And will try to delete it again at browser process exit.